### PR TITLE
Rename variables

### PR DESCRIPTION
**Pull request by joneff**

### CODE REVIEW

File: packages/default/scss/split-button/_layout.scss [hunk 1]
```diff
@@ -9,15 +9,15 @@
         flex: none;
 
         &.k-button-sm {
-            padding: $kendo-split-button-arrow-padding-y-sm $kendo-split-button-arrow-padding-x-sm;
+            padding: $kendo-split-button-arrow-sm-padding-y $kendo-split-button-arrow-sm-padding-x;
```
Comment (TeyaVes) [inline packages/default/scss/split-button/_layout.scss:12]: kendo-split-button-sm-arrow-padding-y here and below

File: packages/default/scss/table/_variables.scss [hunk 2]
```diff
@@ -29,35 +29,35 @@ $kendo-table-cell-padding-x: null !default;
...
+$kendo-table-sm-cell-padding-y: k-map-get( $kendo-spacing, 1 ) !default;
 
-$kendo-table-cell-padding-x-md: k-map-get( $kendo-spacing, 3 ) !default;
-$kendo-table-cell-padding-y-md: k-map-get( $kendo-spacing, 2 ) !default;
+$kendo-table-md-cell-padding-x: k-map-get( $kendo-spacing, 3 ) !default;
+$kendo-table-md-cell-padding-y: k-map-get( $kendo-spacing, 2 ) !default;
 
-$kendo-table-cell-padding-x-lg: k-map-get( $kendo-spacing, 2 ) !default;
-$kendo-table-cell-padding-y-lg: k-map-get( $kendo-spacing, 2.5 ) !default;
+$kendo-table-cell-lg-padding-x: k-map-get( $kendo-spacing, 2 ) !default;
```
Comment (TeyaVes) [inline packages/default/scss/table/_variables.scss:38]: kendo-table-lg-cell..

**Review by TeyaVes — COMMENTED** [review]
are we leaving the base-bg, -text, border and the $shadow-size vars without a kendo prefix?

File: packages/fluent/scss/action-sheet/_variables.scss [hunk 3]
```diff
@@ -88,10 +88,10 @@ $kendo-actionsheet-header-shadow: none !default;
 
 /// Font size of the action sheet subtitle.
 /// @group action-sheet
-$kendo-actionsheet-subtitle-font-size: var( --kendo-font-size-sm, inherit ) !default;
+$kendo-actionsheet-subtitle-font-size: var( --kendo-sm-font-size, inherit ) !default;
```
Comment (Juveniel) [inline packages/fluent/scss/action-sheet/_variables.scss:91]: --kendo-font-size-sm

File: packages/fluent/scss/card/_layout.scss [hunk 4]
```diff
@@ -278,8 +278,8 @@
 
         > .k-card-deck {
             // hide scrollbar
-            margin-block-end: -$default-scrollbar-width;
-            padding-block-end: $default-scrollbar-width;
+            margin-block-end: - $kendo-scrollbar-width;
```
Comment (Juveniel) [inline packages/fluent/scss/card/_layout.scss:281]: there is a redundant space here

File: packages/fluent/scss/core/_variables.scss [hunk 5]
```diff
@@ -111,16 +111,16 @@ $kendo-zindex-loading: 100 !default;
 
         --kendo-font-size: #{$kendo-font-size};
         --kendo-font-size-xs: #{$kendo-font-size-xs};
-        --kendo-font-size-sm: #{$kendo-font-size-sm};
-        --kendo-font-size-md: #{$kendo-font-size-md};
-        --kendo-font-size-lg: #{$kendo-font-size-lg};
+        --kendo-sm-font-size: #{$kendo-font-size-sm};
+        --kendo-md-font-size: #{$kendo-font-size-md};
+        --kendo-lg-font-size: #{$kendo-font-size-lg};
```
Comment (Juveniel) [inline packages/fluent/scss/core/_variables.scss:116]: naming

**Review by Juveniel — COMMENTED** [review]
please check all occurrences of the above mentioned issues

**Review by Juveniel — APPROVED** [review]

**Pull request merged**